### PR TITLE
update readme.md for gradle

### PR DESCRIPTION
**Pull request by ktarbet**

### CODE REVIEW

**Comment by ktarbet** [comment]
I don't think the arguments are being passed, from gradle, such as 

```-Dno.docs=true```

File: README.md [hunk 1]
```diff
@@ -50,24 +50,36 @@ can be scary we would like to encourage you to join us anyways.
...
+`./gradle distTar`
+
+output is here:
+
+opendcs/install/build/distributions/opendcs-main.99.main-SNAPSHOT.zip
 
 To get a simple baseline environment going:
 
-`ant run`
+`./gradle runApp`
```
Comment (MikeNeilson) [inline README.md:82]:
```suggestion
`./gradlew runApp --info`
```

File: README.md [hunk 2]
```diff
@@ -84,15 +96,15 @@ ant test
 # programmed script the task may return failure.
 
 # Test the GUI (NOTE: leave your hands off the keyboard and mouse or the runner gets confused.)
-ant gui-test -Dno.docs=true
+./gradle test -Dno.docs=true
```
Comment (MikeNeilson) [inline README.md:99]:
```suggestion
./gradlew test -Pno.docs=true
```

Comment (MikeNeilson) [reply README.md:99]: on windows it should be -P"no.docs=true"

File: README.md [hunk 3]
```diff
@@ -84,15 +96,15 @@ ant test
...
 
 # Tests of a "live" system.
-ant integration-test -Dno.docs=true -Dopendcs.test.engine=OpenDCS-XML
+./gradle testing:opendcs-tests:test -Dno.docs=true -Dopendcs.test.engine=OpenDCS-XML
```
Comment (MikeNeilson) [inline README.md:102]:
```suggestion
./gradlew testing:opendcs-tests:test -Pno.docs=true -Popendcs.test.engine=OpenDCS-XML
```

File: README.md [hunk 4]
```diff
@@ -84,15 +96,15 @@ ant test
...
 # and if you have docker
-ant integration-test -Dno.docs=true -Dopendcs.test.engine=OpenDCS-Postgres
+gradle testing:opendcs-tests:test -Dno.docs=true -Dopendcs.test.engine=OpenDCS-Postgres
```
Comment (MikeNeilson) [inline README.md:104]:
```suggestion
./gradlew testing:opendcs-tests:test -Pno.docs=true -Popendcs.test.engine=OpenDCS-Postgres
```

File: README.md [hunk 5]
```diff
@@ -104,7 +116,7 @@ This is a current limitation of the ant build.
 To run a specific test only use:
 
 ```
-ant <test target> -Dtests=<Test class name>
+?? <test target> -Dtests=<Test class name>
```
Comment (MikeNeilson) [inline README.md:119]:
```suggestion
./gradlew <test target> --tests # See [gradle documentation ](https://docs.gradle.org/current/userguide/java_testing.html#simple_name_pattern) for more detail
```

File: README.md [hunk 6]
```diff
@@ -121,10 +133,14 @@ If you wish to help in this way please created additional targets and required f
 
 To create the appropriate project files for VS Code or Eclipse you can run the following task:
 
-`ant eclipse-ide-files`
+`./gradle opendcs:eclipse`
```
Comment (MikeNeilson) [inline README.md:136]:
```suggestion
`./gradlew opendcs:eclipse`

Or you can use the VS Code gradle integration.
```

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**